### PR TITLE
things 14, 16 and 18 available for review

### PR DESCRIPTION
**Pull request by rowan-ardc**

### CODE REVIEW

**Comment by ragamouf** [comment]
er - I made edits to your thing 14 over in your branch...! Sorry! May need to re do the commit?

**Comment by ragamouf** [comment]
Hi @rowan-ardc, the link on Thing 16 for Dryad goes to a wiki page on the Joint Data ARchiving Policy. Is this still current and useful to link to? I think Dryad's business model has changed significantly since 23 things, so there might be another more appropriate page to link to?

**Comment by rowan-ardc** [comment]
Hi @ragamouf - I replaced the dryad link with a pointer to the Elsevier policy.  Any further thoughts on this or suggested alternatives?
> 
> 
> Hi @rowan-ardc, the link on Thing 16 for Dryad goes to a wiki page on the Joint Data ARchiving Policy. Is this still current and useful to link to? I think Dryad's business model has changed significantly since 23 things, so there might be another more appropriate page to link to?

**Comment by rowan-ardc** [comment]
Thanks @ragamouf  - looks fine
> 
> 
> er - I made edits to your thing 14 over in your branch...! Sorry! May need to re do the commit?

**Review by mpfl — CHANGES_REQUESTED** [review]
Could you please add links to the previous thing? Sorry, my example in Thing 1 didn't have this because there is no Thing 0!

**Comment by rowan-ardc** [comment]
righto - have fixed navigation
> 
> 
> Could you please add links to the previous thing? Sorry, my example in Thing 1 didn't have this because there is no Thing 0!

**Comment by rowan-ardc** [comment]
Removed credly text & badge files

**Review by mpfl — APPROVED** [review]

File: things/thing-14.md [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+
+**Consider:** why do you think there is an exclamation mark in the
+previous sentence about the publication type for research data?
+
+Do you have a question?  Want to share a resource?
+- Post to the [Data Librarians Slack group](https://tiny.cc/data-librarians) to connect with the community.
+- Tweet to [@ardc_au](https://twitter.com/ARDC_AU) using hashtag #23things
+
+Go to [Thing 15](thing-15.md "thing 15")
+Data Management Plans [All Things](index.md)
```
Comment (NicholaBurton) [inline things/thing-14.md]: need an "or to" before All Things (here and throughout)

File: things/thing-16.md [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+
+-   [National Science
+    Foundation](http://www.nsf.gov/bfa/dias/policy/dmp.jsp) in the US,
+    and note the Data Management Plan requirement (see [Thing 15:
+    DMPs](thing-16.md))
+-   [Wellcome
+    Trust](https://wellcome.ac.uk/funding/managing-grant/policy-data-management-and-sharing)
+    in the UK (1 page)
+
+Now, imagine it is 2020… **consider** what you think Australian research
```
Comment (NicholaBurton) [inline things/thing-16.md]: Pretty soon this won't take much imagination! We might need to update the date... 2025?

File: things/thing-18.md [hunk 3]
```diff
@@ -0,0 +1,177 @@
...
+
+**Consider:** what are some ideas you may have for conversation
+starters?
+
+Do you have a question?  Want to share a resource?
+- Post to the [Data Librarians Slack group](https://tiny.cc/data-librarians) to connect with the community.
+- Tweet to [@ardc_au](https://twitter.com/ARDC_AU) using hashtag #23things
+
+Go to [Thing 19](thing-19.md "thing 19")
+Exploring APIs andApps or to [All Things](index.md)
```
Comment (NicholaBurton) [inline things/thing-18.md]: missing space in "and Apps" here and throughout

**Review by NicholaBurton — CHANGES_REQUESTED** [review]
I notice that these Things are missing the Thing Theme overview that is at the bottom of some of the others (i.e. Things 16 and 18 are part of the theme "Let's talk data"). Should we add this section to all things for consistency?

**Comment by rowan-ardc** [comment]
Reflected in Matthias's example.  I've reinstated to 14, 16 and 18.

> 
> 
> I notice that these Things are missing the Thing Theme overview that is at the bottom of some of the others (i.e. Things 16 and 18 are part of the theme "Let's talk data"). Should we add this section to all things for consistency?

File: things/thing-14.md (hunk 1 above)
Comment (rowan-ardc) [reply things/thing-14.md]: Fixed each occurrence

File: things/thing-16.md (hunk 2 above)
Comment (rowan-ardc) [reply things/thing-16.md]: Changed date to 2025

File: things/thing-18.md (hunk 3 above)
Comment (rowan-ardc) [reply things/thing-18.md]: Fixed each occurrence

**Comment by rowan-ardc** [comment]
requested changes complete

**Pull request merged**